src/condor_daemon_core.V6/daemon_core_main.cpp

```diff
@@ -818,8 +818,11 @@ DaemonCore::kill_immediate_children() {
 			dprintf(D_FULLDEBUG, "Daemon exiting before reaping child pid %d\n", pid_entry->pid);
 			continue;
 		}
+		if (pid_entry->cleanup_signal == 0) {
+			continue;
```

Comment:
Is it worth noting in the log here that the daemon is exiting but deliberately not killing one of its children?

Reply:
I think a message at D_FULLDEBUG would be useful.